.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,32 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: 'State: Unconfirmed, Type: Bug'
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
```

Comment:
Requesting that the user attach the log files (Saros log and possibly IDE log) of a session in which the error appeared would be very important in my opinion.

Reply:
Where can I find the IntelliJ logs and the Saros/I logs?

Reply:
The different log locations are described [here](https://www.saros-project.org/releases/saros-i_0.1.0.html#report-a-bug).

Reply:
I added the information, but I am not sure whether the template became a bit crowded with information and we should rather link this information.